src/shared/components/ShareDropdown/ShareDropdown.tsx

```diff
@@ -0,0 +1,89 @@
+import { Dropdown, DropdownButton, DropdownContent } from '@meemoo/react-components';
+import { Button, IconName, Tabs } from '@viaa/avo2-components';
+import React, { FC, useState } from 'react';
+
+import { useTabs } from '../../hooks';
+import useTranslation from '../../hooks/useTranslation';
+import ShareWithColleagues from '../ShareWithColleagues/ShareWithColleagues';
+import {
+	ShareUserInfo,
+	ShareUserInfoRights,
+} from '../ShareWithColleagues/ShareWithColleagues.types';
+
+import './ShareDropdown.scss';
+
+type ShareDropdownProps = {
+	users: ShareUserInfo[];
+	onAddNewUser: (info: Partial<ShareUserInfo>) => void;
+	onEditRights: (info: ShareUserInfo, newRights: ShareUserInfoRights) => void;
+	onDeleteUser: (info: ShareUserInfo) => void;
+};
+const ShareDropdown: FC<ShareDropdownProps> = ({
+	users,
+	onAddNewUser,
+	onEditRights,
+	onDeleteUser,
+}) => {
+	const { tText } = useTranslation();
+	const [isShareDropdownOpen, setIsShareDropdownOpen] = useState<boolean>(false);
+	const [tab, setActiveTab, tabs] = useTabs(
+		[
+			{
+				id: 'colleagues',
+				label: tText('shared/components/share-dropdown/share-dropdown___collegas'),
+				icon: IconName.userTeacher,
+			},
+			{
+				id: 'pupils',
+				label: tText('shared/components/share-dropdown/share-dropdown___leerlingen'),
+				icon: IconName.userStudent,
+			},
+		],
+		'colleagues'
+	);
+
+	const handleShareButtonClicked = () => {
+		setIsShareDropdownOpen(!isShareDropdownOpen);
+	};
+
+	return (
+		<Dropdown
+			isOpen={isShareDropdownOpen}
+			onClose={() => setIsShareDropdownOpen(false)}
+			className="c-share-dropdown"
```

Comment:
You can add an option to show the dropdown right aligned to the button position:
```
placement="bottom-end"
```

![image](https://github.com/viaacode/avo2-client/assets/1710840/e09eeb50-b219-4688-9a50-2f1a3221e36a)


so it matches the design:
https://www.figma.com/file/xWFZmBGjGetTJOlIqnaeEY/Delen-%26-Samenwerken-(MASTER-%E2%80%94-Internal-use-only)?type=design&node-id=1037-7792&t=e4WArnelsp31sfCr-0

Reply:
I have fixed  this in another branch